oshi-core/src/test/java/oshi/util/TupleTest.java

```diff
@@ -47,23 +47,22 @@ public void testTuples() {
         Quintet<String, Integer, Long, Character, BigInteger> quintet = new Quintet<>("D", 4, Long.valueOf("0"), 'd',
                 BigInteger.ZERO);
 
-        assertEquals("A", pair.getA());
-        assertEquals("B", triplet.getA());
-        assertEquals("C", quartet.getA());
-        assertEquals("D", quintet.getA());
+        assertEquals(pair.getA()+" should be equal to A","A", pair.getA());
```

Comment:
JUnit actually does include the "expected" and "actual" values for an equality test so your output on a failure would be redundant with that, and not help.  On failure, I might get the message "B should be equal to A" and it wouldn't let me find the line.  Better to have the string "Pair.getA() should be A".

Reply:
Thank you for the feedback, I will work on the review.